server/sockets/list.js

```diff
@@ -452,6 +452,60 @@ const changeListType = (socket, dashboardClients, cohortClients, listClients) =>
       });
   });
 
+const removeMember = (socket, dashboardClients, listClients, cohortClients) =>
+  socket.on(ListActionTypes.REMOVE_MEMBER_SUCCESS, data => {
+    const { listId, userId } = data;
+
+    // Broadcast to every user on the list
+    socket.broadcast
+      .to(`sack-${listId}`)
+      .emit(ListActionTypes.REMOVE_MEMBER_SUCCESS, data);
+
+    // Broadcast to that removed user if he is at dashboard view
+    if (dashboardClients.has(userId)) {
```

Comment:
we discussed checking clients size... Did we decide to no check the client's map size?